### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by matthewdeanmartin** [FHPythonUtils/PyTaintX#1]
Looks like global search and replace of pyt clobbered file types.

This is the pypi packaged versions.

```
FileNotFoundError: [Errno 2] No such file or directory: '...\.virtualenvs\\search_service-3tw2h1cf\\lib\\site-packages\\pytaintx\\vulnerability_definitions\\all_trigger_words.pytaintx'

$ ls ...//.virtualenvs//search_service-3tw2h1cf//lib//site-packages//pytaintx//vulnerability_definitions//
all_trigger_words.pyt  blackbox_mapping.json  django_trigger_words.pyt  
flask_trigger_words.pyt  README.rst  test_positions.pyt  test_triggers.pyt
```

**Comment by FredHappyface** [comment]
Thanks for the report. I'm certainly regretting that now. I'll take a look at it and hopefully get a fix out in the near future

**Comment by FredHappyface** [comment]
Fixed in 0.5.1. Thanks again for the report
